inception-search-mtas/src/main/java/de/tudarmstadt/ukp/inception/search/index/mtas/MtasUimaParser.java

```diff
@@ -203,107 +208,150 @@ else if (annotation instanceof Sentence) {
                     mtasSentence.setOffset(annotation.getBegin(), annotation.getEnd());
                     mtasSentence.addPositionRange(beginToken, endToken);
                     tokenCollection.add(mtasSentence);
-                }
-                else {
+                } else {
                     // Other annotation types - annotate the features
                     if (layers.get(annotationName) != null) {
-                        // Add the UI annotation name to the index as an annotation.
+                        // Add the UI annotation name to the index as an
+                        // annotation.
                         // Replace spaces with underscore in the UI name.
-                        
+
                         MtasToken mtasAnnotation = new MtasTokenString(mtasId++,
                                 annotationUiName.replace(" ", "_") + MtasToken.DELIMITER,
                                 beginToken);
                         mtasAnnotation.setOffset(annotation.getBegin(), annotation.getEnd());
                         mtasAnnotation.addPositionRange(beginToken, endToken);
                         tokenCollection.add(mtasAnnotation);
 
-                        // Get features for this annotation, if it is indexed. First comes the
+                        // Get features for this annotation, if it is indexed.
+                        // First comes the
                         // internal feature name, then the UI feature name
-                        for (AnnotationFeature feature: layerFeatures.get(annotationName)) {
+                        for (AnnotationFeature feature : layerFeatures.get(annotationName)) {
```

Comment:
I would put this code into a function, it is nested really deep. It would also make it more readable by giving it a good function name then.

Reply:
I don't see the need to create a function here.